micro_dl/plotting/plot_utils.py

```diff
@@ -74,6 +76,98 @@ def save_predicted_images(input_batch, target_batch, pred_batch,
         plt.close(fig)
 
 
+def plot_xyz(image_dir,
+             pos_idx,
+             fig_name,
+             mean_std=None,
+             tol=1,
+             font_size=15,
+             color_map='gray',
+             fig_title=None,
+             margin=20,
+             z_scale=5,
+             channel_str=None):
+    """
+    Given an image directory, loads a z-stack, plots the center sections of
+    xy, yz and xz.
+
+    :param str image_dir: Directory containing z-stacks
+    :param int pos_idx: Which FOV to plot
+    :param str fig_name: Full path of where to write figure file
+    :param tuple mean_std: If None, just assume the image will plot well as is,
+        if tuple containing a mean and std (e.g. mean over training data),
+        set z-stack mean and std and convert to uint16
+    :param float tol: top and bottom % of intensity to saturate (hist clipping)
+    :param int font_size: font size of the image title
+    :param str color_map: Matplotlib colormap
+    :param str fig_title: Figure title
+    :param int margin: Number of pixel margin between xy and xz, yz plots
+    :param int z_scale: How much to upsample in z (to be able to see xz and yz)
+    :param str channel_str: If there's more than one channel in image_dir
+        (e.g. input image dir as opposed to predictions) use this str to select
+        which channel to build z-stack from
+    """
+
+    search_str = os.path.join(image_dir, "*p{:03d}*".format(pos_idx))
+    slice_names = natsort.natsorted(glob.glob(search_str))
+
+    if channel_str is not None:
```

Comment:
guess one could pass channel_ids as channel_str here? maybe specify what this str could be in doc string?